test/backend/test_node.py

```diff
@@ -343,6 +343,58 @@ def test_conv(self):
 
     np.testing.assert_almost_equal(output["Y"], test_output, decimal=5)
 
+  def test_conv_integer(self):
+    if legacy_opset_pre_ver(10):
+      raise unittest.SkipTest(
+          "ONNX version {} doesn't support ConvInteger.".format(
+              defs.onnx_opset_version()))
+
+    # Test w_zero_point
+    x = np.array([2, 3, 4, 5, 6, 7, 8, 9, 10]).astype(np.int8).reshape((1, 1, 3,
+                                                                        3))
+    w = np.array([2, 2, 2, 2]).astype(np.int8).reshape((1, 1, 2, 2))
+    w_zero_point = np.int8(1)
+    y = np.array([16, 20, 28, 32]).astype(np.int32).reshape((1, 1, 2, 2))
+
+    node = helper.make_node(
+        "ConvInteger", ["X", "W", "w_zero_point"], ["Y"],
+        kernel_shape=[2, 2],
+        pads=[0, 0, 0, 0],
+        dilations=[1, 1])
+    output = run_node(node, [x, w, w_zero_point])
+    np.testing.assert_almost_equal(output["Y"], y)
+
+    # Test x_zero_point and w_zero_point
+    x = np.array([2, 3, 4, 5, 6, 7, 8, 9, 10]).astype(np.int8).reshape((1, 1, 3,
+                                                                        3))
+    x_zero_point = np.int8(1)
+    w = np.array([2, 2, 2, 2]).astype(np.int8).reshape((1, 1, 2, 2))
+    w_zero_point = np.int8(1)
+    y = np.array([12, 16, 24, 28]).astype(np.int32).reshape((1, 1, 2, 2))
+
+    node = helper.make_node(
+        "ConvInteger", ["X", "W", "x_zero_point", "w_zero_point"], ["Y"],
+        kernel_shape=[2, 2],
+        pads=[0, 0, 0, 0],
+        dilations=[1, 1])
+    output = run_node(node, [x, w, x_zero_point, w_zero_point])
+    np.testing.assert_almost_equal(output["Y"], y)
+
+    # Test w_zero_point as 1d tensor
+    x = np.array([2, 3, 4, 5, 6, 7, 8, 9, 10]).astype(np.int8).reshape((1, 1, 3,
+                                                                        3))
+    w = np.array([2, 2, 2, 2]).astype(np.int8).reshape((1, 1, 2, 2))
+    w_zero_point = np.array([1]).astype(np.int8)
```

Comment:
maybe modify w and w_zero_point to have multiple feature maps(M) to cover more use case